go.mod

```
@@ -114,15 +114,15 @@ require (
 	golang.org/x/sys v0.0.0-20210426230700-d19ff857e887
 	google.golang.org/grpc v1.33.2
 	gopkg.in/yaml.v2 v2.4.0
-	k8s.io/api v0.19.14
-	k8s.io/apimachinery v0.19.14
-	k8s.io/apiserver v0.19.14
+	k8s.io/api v0.19.15
+	k8s.io/apimachinery v0.19.15
+	k8s.io/apiserver v0.19.15
 	k8s.io/client-go v11.0.1-0.20190409021438-1a26190bd76a+incompatible
-	k8s.io/cloud-provider v0.19.14
-	k8s.io/component-base v0.19.14
+	k8s.io/cloud-provider v0.19.15
+	k8s.io/component-base v0.19.15
 	k8s.io/cri-api v0.20.1
```

This one is part of the replaced k8s.io submodule set as well
```suggestion
	k8s.io/cri-api v0.19.15
```

---

If I change  that and run `go mod vendor` comes back to 0.20.1

---

Nevermind, containerd's CRI stuff wants this pulled forward so it's fine.